manifests/tigera-operator.yaml

```diff
@@ -5,6 +5,28 @@ metadata:
   labels:
     name: tigera-operator
 ---
+# Source: crds/calico/_.yaml
```

Comment:
I think this shouldn't be here - probably generated by mistake? looks like a stray `_.yaml` file somewhere